styleguide/source/_patterns/03-templates/04-topic/_topic.scss

```diff
@@ -23,7 +23,11 @@
 .topic .grid-region {
 
   .grid-region_content:not(:first-child){
-    @include gutter($padding-top-full...);
+    @include gutter($padding-bottom-full...);
```

Comment:
![pattern_lab_-_templates-topic](https://user-images.githubusercontent.com/238201/33138525-558e8a4a-cf70-11e7-9da3-6f457ceec59c.png)
This breaks the spacing between the first and second blocks. If you're switching to bottom gutters then maybe the selector should be `:not(:last-child)`?

Reply:
yeah. probably. will change